ui-participant/src/hub/HubRouter.tsx

```diff
@@ -34,6 +35,8 @@ export default function HubRouter() {
       <Route path="profile/:ppUserId?"
         element={<ParticipantProfile/>}/>
       <Route path="kits"
+        element={<KitsPage/>}/>
+      <Route path="kits/in_person"
```

Comment:
FWIW, it's generally best practice to use dashes rather than underscores in URLs.  They're easier to see in links, and apparently search engines like them better